src/Controller/DjDashboardController.php

```diff
@@ -2,18 +2,37 @@
 
 namespace App\Controller;
 
+use App\Entity\Reservation;
+use App\Repository\ArtistRepository;
+use App\Repository\ReservationRepository;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Request as HttpFoundationRequest;
 use Symfony\Component\HttpFoundation\RequestStack;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;
 
 #[Route('/espace-dj', name: 'dashboard_dj_')]
 class DjDashboardController extends AbstractController
 {
+    public const MAX_ELEMENTS = 3;
+
     #[Route('/', name: 'index')]
-    public function index(): Response
-    {
-        return $this->render('dj_dashboard/index.html.twig');
+    public function index(
+        ReservationRepository $reservationRepo,
+        AuthenticationUtils $authenticationUtils,
+        ArtistRepository $artistRepository
+    ): Response {
+        $emailArtist = $authenticationUtils->getLastUsername();
+        $artist = $artistRepository->findOneBy(['email' => $emailArtist]);
```

Comment:
c'est pas plutot $this->getUser() que tu veux faire ?

Reply:
Avec le problème de circular référence, j'ai dû privilégier qui dépend de qui, et du coup dans les fixtures, l'user n'a pas d'id artist, donc je ne suis pas certaine que ça marche. Mais comme ils ont le même email, ça, ça marche.

Reply:
mais tu peux faire une bidirectionnelle et donc quand meme faire un $this->getUser()->getArtist() du coup
tu n'as dans tous les cas pas à utilser AuthenticationUtils dans le controller directement